File: desktop/src/feedback/dests.js
// The screens the feedback sweep photographs — a CommonJS twin of frontend/src/lib/dests.js (the
// renderer's ESM list; Electron's main process cannot require it). Pinned equal by
// desktop/test/feedback-dests-sync.test.mjs, together with the opener's allow-list.
'use strict'
const DESTS = [
  { id: 'board', section: 'Board', sub: null },
  { id: 'strategy-hold', section: 'Strategy', sub: 'hold' },
  { id: 'strategy-arbitrage', section: 'Strategy', sub: 'arbitrage' },
  { id: 'economy-inflation', section: 'Economy', sub: 'inflation' },
  { id: 'economy-market', section: 'Economy', sub: 'market' },
  { id: 'trading-workspace', section: 'Trading', sub: 'workspace' },
  { id: 'trading-live', section: 'Trading', sub: 'live' },
  { id: 'trading-sales', section: 'Trading', sub: 'sales' },
  { id: 'settings', section: 'Settings', sub: null },
]
module.exports = { DESTS }

File: ops/feedback-bot/opener/dests.py
"""The screens a report may carry — the opener's allow-list. Output filenames derive from THIS
tuple, never from the report; pinned to frontend/src/lib/dests.js by desktop/test/feedback-dests-sync.test.mjs."""
SCREENS = (
    "current",
    "board",
    "strategy-hold",
    "strategy-arbitrage",
    "economy-inflation",
    "economy-market",
    "trading-workspace",
    "trading-live",
    "trading-sales",
    "settings",
)
